nixos/modules/services/networking/nftables.nix

```diff
@@ -294,7 +307,11 @@ in
             '') enabledTables)}
             ${cfg.ruleset}
             ${lib.optionalString (cfg.rulesetFile != null) ''
-              include "${cfg.rulesetFile}"
+              ${if cfg.rulesetFileFlatten then
```

Comment:
This [string interpolation](https://nixos.org/manual/nix/stable/language/string-interpolation#string-interpolation) can be removed (together with the outer `''`).

Reply:
thx, changed